home/views.py

```diff
@@ -12,19 +12,33 @@ def landingpage(request):
     return render(request, 'home/landingpage.html')
 
 
-def displayQuestion(request, **kwargs):
+def displayQuestionPage(request, **kwargs):
+    form = CommentForm(request.POST)
     question = Question.objects.all().get(id=kwargs['pk'])
     sortAnsBy = request.GET["sortanswersby"] if 'sortanswersby' in request.GET else ''
     context = {
         "question": question,
         "answers": question.get_answers_feed(sortAnsBy),
         "answersCount": question.answer_set.count(),
         "tags": question.tags.values(),
-        "title": question.get_question_title()
+        "title": question.get_question_title(),
+        "form": form
     }
+    if request.method == 'POST':
+        return addAnswer(form, request.profile, question)
```

Comment:
not sure moving the code out to a function makes sense, and the risk is that people migt think that helper function is a view function since its in the views file.

The function name is also not Pythonic.

Reply:
fixed